cpp/include/cudf/io/parquet.hpp

```diff
@@ -1299,6 +1358,24 @@ class chunked_parquet_writer_options_builder {
     return *this;
   }
 
+  /**
+   * @brief Sets the desired maximum size in bytes for min and max values in the column index.
+   * Values exceeding this limit will be truncated, but modified such that they will still
+   * be valid lower and upper bounds. This only applies to variable length types, such as string.
+   * Maximum values will not be truncated if there is no suitable truncation that results in
+   * a valid upper bound.
```

Comment:
This is not "brief". This is "long".

Please cut off real brief (first sentence) and the rest.